src/components/main/DeleteItemDialog.tsx

```diff
@@ -20,17 +20,33 @@ const descriptionId = 'alert-dialog-description';
 
 const { DELETE_ITEMS, DELETE_ITEM } = MUTATION_KEYS;
 
-const DeleteItemDialog = ({ itemIds, open, handleClose }) => {
+type Props = {
+  open?: boolean;
+  handleClose: () => void;
+  itemIds: string[];
+};
+
+const DeleteItemDialog: FC<Props> = ({
+  itemIds,
+  open = false,
+  handleClose,
+}) => {
   const { t: translateBuilder } = useBuilderTranslation();
 
-  const { mutate: deleteItems } = useMutation(DELETE_ITEMS);
-  const { mutate: deleteItem } = useMutation(DELETE_ITEM);
+  const { mutate: deleteItems } = useMutation<unknown, unknown, string[]>(
+    DELETE_ITEMS,
+  );
+  const { mutate: deleteItem } = useMutation<unknown, unknown, string>(
+    DELETE_ITEM,
+  );
 
   const onDelete = () => {
     if (itemIds.length > 1) {
       deleteItems(itemIds);
     } else {
-      deleteItem(itemIds);
+      // we still use this call because it has feedback
+      // remove when deleteItems will have feedback
+      deleteItem(itemIds as unknown as string);
```

Comment:
Wouldn't it be better to send `itemIds[0]` instead ?

Reply:
Actually it's either an array or a string.. not the best I know D: